package.json

```diff
@@ -32,8 +31,12 @@
     "execa": "^5.1.1",
     "jest": "^27.2.5",
     "prettier": "^2.4.1",
+    "tsd": "^0.18.0",
     "typescript": "^4.4.4"
   },
+  "peerDependencies": {
+    "tsd": ">=0.15.0"
```

Comment:
It felt like `peerDependencies` is good idea. `v0.15` is the minimal version with necessary API.